src/components/Select/Select.jsx

```diff
@@ -396,7 +396,7 @@ export default class Select extends React.Component {
       return customButton(this.onButtonClick, this.onSearchChange);
     }
     if (customButton) {
-      return customButton(this.onButtonClick);
+      return customButton(this.onButtonClick, this.state.isOpen);
```

Comment:
Just checking here @gmzjuliana, but does this signature for customButton look different to the one 3 lines up, or does it always return the open state as a second parameter?

I'm just wondering as it looks like theres potentially 2 signatures that the customButton function could have.

`customButton(function, function)` or `customButton(function, bool)`

and if you are passing in the first type, and don't do the check for types, you might end up with expectations that you are returning a boolean. So it could potentially break existing code. It might be nicer to have something like.

`customButton(function, function, bool)` and you can provide null as the second argument. 

This might not be an issue at all, just thought i'd ask the question at least 😉 Thanks for doing this work, its looking great 
 👍